api/v1/awsaccount_types.go

```diff
@@ -33,34 +33,27 @@ type AwsAccountSpec struct {
 	Zones    []string `json:"zones"`
 }
 
-// +kubebuilder:validation:Enum=creating user;creating login profile;creating access key;created
-type AccountStatus string
-
-const (
-	CreatingUser AccountStatus = "creating user"
-
-	CreatingLoginProfile AccountStatus = "creating login profile"
-
-	CreatingAccessKey AccountStatus = "creating access key"
-
-	Created AccountStatus = "created"
-)
-
 // AwsAccountStatus defines the observed state of AwsAccount
 type AwsAccountStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// +optional
-	Account AccountStatus `json:"account"`
+	UserCreated bool `json:"userCreated"`
 
 	// +optional
-	UserGroups []string `json:"userGroups"`
+	LoginProfileCreated bool `json:"loginProfileCreated"`
 
 	// +optional
-	ZonesCreated []string `json:"zonesCreated"`
+	AccessKeyCreated bool `json:"accessKeyCreated"`
+
+	// +optional
+	UserGroups []string `json:"userGroups"`
 }
 
+//+kubebuilder:object:root=true
+//+kubebuilder:object:root=true
+//+kubebuilder:object:root=true
 //+kubebuilder:object:root=true
```

Comment:
Is that how it should be?  Or there should be only one?
```suggestion
//+kubebuilder:object:root=true
```